src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -6,11 +6,19 @@
 #include <string.h>
 #include <vector>
 
+#include "icushim.h"
 #include "config.h"
 #include "locale.hpp"
 #include "holders.h"
 #include "errors.h"
 
+#ifndef FEATURE_FIXED_ICU_VERSION
+// The System.Globalization.Native.so is built in mode that allows it to find the local
```

Comment:
But I agree that the comment is somehow convoluted, I'll try to clean it a bit and integrate both points in it.